src/components/ListItem/ListItem.test.js

```diff
@@ -102,4 +103,30 @@ describe("ListItem", () => {
     );
     expect(wrapper).toMatchSnapshot();
   });
+
+  it('should invoke editing when list item is clicked', () => {
```

Comment:
I couldnt get a simulated click to work but i think that was because i was targeting list-item incorrectly. Good job.